schema/mzqc_schema.json

```diff
@@ -23,6 +23,14 @@
                     "description": "Description and comments about the mzQC file contents.",
                     "type": "string"
                 },
+                "contactName": {
+                    "description": "Name of contact a particular mzQC file.",
```

Comment:
`Creator of this mzQC file`

Reply:
Not necessarily creator?

Reply:
I agree, it can be the PI as well (and probably should for long-term availability).